pom.xml

```diff
@@ -5121,6 +5129,16 @@
                 <artifactId>microprofile-metrics-api</artifactId>
                 <version>${version.org.eclipse.microprofile.metrics.api}</version>
             </dependency>
+            <dependency>
+                <groupId>org.eclipse.microprofile.metrics</groupId>
+                <artifactId>microprofile-metrics-api-tck</artifactId>
+                <version>${version.org.eclipse.microprofile.metrics.api}</version>
```

Comment:
I think this should be `<scope>test</scope>`?

Reply:
you're right, I'll fix it. They are in the `test` scope in testsuite/integration/microprofile-tck/metrics/pom.xml but they should also have this scope in the dependencyManagement.